sql-plugin/src/main/311until330-all/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -80,4 +80,15 @@ object GpuTypeShims {
 
   def csvRead(cv: cudf.ColumnVector, dt: DataType): cudf.ColumnVector =
     throw new RuntimeException(s"Not support type $dt.")
+
+  /**
+   * Spark supports interval type from 320, but GPU from 330, so just return false
```

Comment:
I'm missing the context here -- why do we not support this until Spark 3.3 if Spark 3.2 supports it?

Reply:
Spark supports interval to/from Parquet from 330,  and lots of interval operators from 330, so here return false.